src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -500,6 +514,15 @@ internal bool CheckIfIsByRefLikeAttributeShouldBeEmbedded(DiagnosticBag diagnost
                 WellKnownMember.System_Runtime_CompilerServices_IsByRefLikeAttribute__ctor);
         }
 
+        internal bool CheckIfNullableAttributeShouldBeEmbedded(DiagnosticBag diagnosticsOpt, Location locationOpt)
+        {
+            return CheckIfAttributeShouldBeEmbedded(
+                diagnosticsOpt,
+                locationOpt,
+                WellKnownType.System_Runtime_CompilerServices_NullableAttribute,
+                WellKnownMember.System_Runtime_CompilerServices_NullableAttribute__ctor);
```

Comment:
>System_Runtime_CompilerServices_NullableAttribute__ctor [](start = 32, length = 55)

It looks like this attribute should has two constructors. Should we check both of them?

Reply:
Yes. See `AttributeTests_Nullable.ExplicitAttribute_MissingConstructor`.